build.gradle

```diff
@@ -223,6 +223,17 @@ task pushDockerImage {
     }
 }
 
+task testInitNakadi(type: Exec) {
```

Comment:
Can we set env variable inside fullTest/fullAcceptanceTest to not duplicate the code ?